microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -1,147 +1,22 @@
 :_content-type: ASSEMBLY
 [id="microshift-storage-plugin-overview"]
-= MicroShift storage plug-in overview 
+= MicroShift storage plugin overview 
 include::_attributes/common-attributes.adoc[]
 :context: microshift-storage-plugin-overview
 
 toc::[]
 
-{product-title} enables dynamic storage provisioning that is ready for immediate use with the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider. The LVMS plugin is the Red Hat downstream version of TopoLVM, a CSI plug-in for managing LVM volumes for Kubernetes. 
+{product-title} enables dynamic storage provisioning that is ready for immediate use with the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider. The LVMS plugin is the Red Hat downstream version of TopoLVM, a CSI plugin for managing LVM volumes for Kubernetes. 
```

Comment:
Vale is cranky about `plugin` instead of `plug-in`, but this is correct according to the supplementary style guide. I think you're good here.